### PR TITLE
Update explore-intro.html

### PR DESCRIPTION
**Pull request by linweiyuan**
Improve translation

### CODE REVIEW

File: content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h2>Kubernetes Pods</h2>
 <!--
         <p>When you created a Deployment in Module <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, Kubernetes created a <b>Pod</b> to host your application instance. A Pod is a Kubernetes abstraction that represents a group of one or more application containers (such as Docker or rkt), and some shared resources for those containers. Those resources include:</p>
 -->
-        <p>在模块 <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>创建 Deployment 时, Kubernetes 添加 a <b>Pod</b> 托管你的应用实例。Pod 是 Kubernetes 抽象出来的，表示一组一个或多个应用程序容器（如 Docker 或 rkt ），以及这些容器的一些共享资源。这些资源包括:</p>
+        <p>在模块 <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>创建 Deployment 时, Kubernetes 添加了一个 <b>Pod</b> 来托管你的应用实例。Pod 是 Kubernetes 抽象出来的，表示一组一个或多个应用程序容器（如 Docker 或 rkt ），以及这些容器的一些共享资源。这些资源包括:</p>
```
Comment (tanjunchen) [inline content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html:46]: @linweiyuan
Thanks
/LGTM

Comment (tanjunchen) [reply content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html:46]: pls see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement

**Comment by tanjunchen** [comment]
`If you haven't yet, post a comment ("I signed it") to the PR—this is needed for the system to recheck your contribution authorization.
`
/retest

**Comment by linweiyuan** [comment]
I signed it

File: content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html (hunk 1 above)
Comment (linweiyuan) [reply content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html:46]: Done

**Pull request merged**